frontend/simple-mercari-web/src/components/Header/Header.css

```diff
@@ -11,7 +11,8 @@ header {
   width: 100px;
 }
 
-.logo{
   width: 7em;
   aspect-ratio: 7/3.5;
-}
\ No newline at end of file
+  padding-left: 5em;
+}
```

Comment:
Maybe you forgot to fix this syntax error here!

Reply:
hi! Im not sure what the syntax error is? Please let me know

Reply:
I think you're missing an opening bracket, and it didn't compile when I checkout this branch!

Reply:
i think it should be fixed now, sorry about that!